includes/Fragments/NavigationMenuAccessControl.php

```diff
@@ -92,4 +97,46 @@ protected function setupNavMenuAccess($currentUser)
             $this->assign('nav__domainList', $this->getDomainAccessManager()->getAllowedDomains($currentUser));
         }
     }
-}
\ No newline at end of file
+
+    /**
+     * Sets up the badges to draw attention to issues on various admin pages.
+     *
+     * This function checks to see if a user can access the pages, and if so checks the count of problem areas.
+     * If problem areas are found, a number greater than 0 will cause the badge to show up.
+     *
+     * @param User        $currentUser The current user
+     * @param PdoDatabase $database    Database instance
+     *
+     * @return void
+     */
+    public function setUpNavBarBadges($currentUser, $database) {
```

Comment:
Another nitpick - we're on a modern-enough version of PHP that you can do something like this to enforce the types on the function call, which we probably should be doing everywhere we can:
```suggestion
    public function setUpNavBarBadges(User $currentUser, PdoDatabase $database) {
```

Annoyingly, this has to be applied from the the called functions before it can be applied to the calling functions (or done at the same time), which is why it's not been applied above for `setupNavMenuAccess`, but I think everything here can have the type definitions applied.